Modify the docs for the automatic network instrumentation (#37)
undefinedlabs/scope-docs: docs/ios-manual-logging.md

sidebar_label: Manual logging
---

> Scope by default otputs `NSLog`, `print` and `os_log` (stdout and stderr) messages in the Logs, but sometimes those messages are associated to the test Span instead of being associated to the Span where the log was generated. 

Reviewer: ```suggestion
Scope captures `NSLog`, `print` and `os_log` (stdout and stderr) output automatically, and associates them to the active span when they were generated if possible, or to the test span otherwise.
```

> In order to improve the information you can get from Scope, you can also add your custom application logs. It will allow you to contextualize much better the state of your application when you review the Scope output.

Reviewer: ```suggestion
In order for the Scope agent to capture information about the line of source code that generated each log event and use different log levels for later filtering, we recommend adding manual logging instrumentation, as described in the following sections.
```

> In order to add your application logs to Scope, you need to do some changes to your code.

Reviewer: ```suggestion
```

> ## Installation (only needed for custom behaviour)

Reviewer: ```suggestion
## Manual instrumentation

### Installation
```

<!--END_DOCUSAURUS_CODE_TABS-->

> ## Usage (only needed for custom behaviour)

Reviewer: ```suggestion
### Usage
```
